From a73b57996274899bf9141f54897e7e9137b50935 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 28 Feb 2023 18:17:52 +0100
Subject: [PATCH 284/954] feat(stm32mp25-fdts): add eMMC supplies on eval board

Add vmmc-supply and vqmmc-supply properties in sdmmc2 node for eMMC.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git c34382bdcb7b0eaabb4755f35e9dfa64bb992816 feat(stm32mp25-fdts): add eMMC supplies on eval board]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I59bb797dcf57b9e77f26d827507f61e973febb9d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/293073
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 fdts/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index ccfa9f5aca..d29e2ca3eb 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -218,6 +218,8 @@
 	no-sdio;
 	st,neg-edge;
 	bus-width = <8>;
+	vmmc-supply = <&vdd_emmc>;
+	vqmmc-supply = <&vddio2>;
 	status = "okay";
 };
 
-- 
2.53.0

